locale/enUS.lua

```diff
@@ -3,6 +3,9 @@ local fmt = string.format
 local L = LibStub("AceLocale-3.0"):NewLocale(addonName, "enUS", true)
 
 L[addonName] = "Rank Sentinel"
+L["Enable"] = "Enable"
```

Comment:
Were these added back intentionally? Smaller table scan, but less easy/lazy localization.

Reply:
Not intentional, no idea how I got them when I forked and cloned tbh 🗡️